ATM.java

```diff
@@ -0,0 +1,197 @@
+import java.util.Scanner;
+
+public class ATM
+{
+    public static void main(String[] args)
+    {
+        Scanner kb = new Scanner(System.in);
+        CheckingAccount checking = new CheckingAccount();
+        SavingsAccount savings = new SavingsAccount();
+        
+        //initializing the min 10 in the account
+        checking.CheckingAccount(10);
+        savings.SavingsAccount(10);
+        
+        String checkBank = "d, w, c, t, q";
+        boolean stop = false;
+        int decision;
+        int add;
+        int totalCheck;
+        int totalSave;
+        int pick;
```

Comment:
You should be declaring these variables closer to where they are actually used. But you don't need a separate variable for every single input.